pkgs/top-level/python-packages.nix

```diff
@@ -32014,6 +32014,40 @@ EOF
   yenc = callPackage ../development/python-modules/yenc {
   };
 
+  zeep = buildPythonPackage rec {
```

Comment:
Thanks @Mic92 , sorry I didn't realise. I have pushed a new version. And tests now work with 2.7 (but not 3.5 - testtools dep).